desktop/components/main_layout/templates/index.jade

```diff
@@ -9,6 +9,7 @@ html(
   data-useragent=userAgent
   data-user-type=sd.CURRENT_USER && sd.CURRENT_USER.type
   data-lab-features=sd.CURRENT_USER && sd.CURRENT_USER.lab_features && sd.CURRENT_USER.lab_features.join(',')
+  class= sd.HIGHLIGHT_HEADER_SEARCH == 'highlighted' ? 'html-highlight-header-search-split-test' : ''
```

Comment:
Conditionally adding a class here to apply the modified styles for the a/b test.